src/sentry/uptime/models.py

```diff
@@ -374,6 +404,11 @@ def create_detector_from_project_subscription(project_sub: ProjectUptimeSubscrip
             "environment": env,
             "mode": project_sub.mode,
         },
+        workflow_condition_group=condition_group,
+    )
+    DetectorState.objects.create(
```

Comment:
i don't believe you'll need to make or modify this object -- i think it should be encapsulated in the `StatefulDetectorHandler`, but cc @wedamija to confirm.